.github/workflows/publish-latest.yml

```diff
@@ -87,3 +87,33 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  log-failure-metric:
+    # Send data point 1 to metric PublishLatestFailure, if success
+    runs-on: ubuntu-latest
+    environment: ci
+    needs: build-test
```

Comment:
```suggestion
    needs: publish
```

Reply:
I also wonder if we should just remove `needs: publish` line and only have `if: ${{ failure() }}`

This will ensure that this runs when even when jobs before `publish` has failed.

Reply:
so `if: ${{ failure() }}` catches any failure from `build-test` and `publish`, whatever in this workflow?

If that's the case, yea, let's remove `needs` 👍

Reply:
Done!